misc/python/materialize/cloudtest/config/environment_config.py

```diff
@@ -90,7 +90,7 @@ def load_environment_config(pytestconfig: pytest.Config) -> EnvironmentConfig:
         environment_context=args.environment_context,
         controllers=controllers,
         e2e_test_user_email=args.e2e_test_user_email,
-        refresh_auth_fn=lambda: get_auth(args),
+        refresh_auth_fn=None,  # type: ignore
```

Comment:
Why has this changed?

Reply:
Oh, this is in `environment_config`. This file won't any longer exist after a rebase...

(I think I have anyway reworked that in the cloud repo.)

Reply:
Let me see where that might line up after rebasing, it seems to be still there